dataland-frontend/tests/e2e/fixtures/FixtureUtils.ts

```diff
@@ -27,6 +27,10 @@ export function generateFixtureDataset<T>(
       t: data,
       reportingPeriod: reportingPeriodGenerator(data),
     });
+    if (id > 1) {
+      fixtureDataset[id - 1].companyInformation.parentCompanyLei =
+        fixtureDataset[id - 2].companyInformation.identifiers["Lei"]?.[0] ?? null;
+    }
```

Comment:
magic numbers?

Reply:
I use the previously set the previously created company as parent company (if the company has a LEI). The index of the loop starts with 1 (and not 0 for some reason).

Reply:
magic numbers 1 and 0 are not a problem (2 is also fine)